Formula/ceph-client.rb

```diff
@@ -0,0 +1,103 @@
+class CephClient < Formula
+  desc "Rados and RBD CLIs and libraries of the Ceph project"
```

Comment:
s/Rados/RADOS/.  A simpler description like "Ceph client tools and libraries" might work too, for any users who have heard of Ceph but aren't familiar with the concept of RADOS.